src/main/java/Event.java

```diff
@@ -0,0 +1,42 @@
+//package duke;
+/**
+ * Event <- Task
+ */
+public class Event extends Task {
+    String description;
+    String time;
+    boolean isDone;
```

Comment:
I like how your class member names are clear and adhered to the Java coding standards. All your class variables naming are in camelCase which is good!